Actions/AddExistingApp/action.yaml

```diff
@@ -13,7 +13,7 @@ inputs:
     required: false
     default: ${{ github.token }}
   parentTelemetryScopeJson:
-    description: Specifies the parent telemetry scope for the Telemetry signal
+    description: Specifies the parent telemetry scope for the telemetry signal
     required: false

```

Comment:
this needs to be true if you change the .ps1 required to be true - and then the default should be removed.

Reply:
Rethinking about it I believe it shouldnt be mandatory so Ill fix it accordingly